float32[] object_distance
float32[] object_x
float32[] object_y
int32[] classId
float32[] confidence
float32[] middle_lane